pkgs/top-level/all-packages.nix

```diff
@@ -33613,7 +33613,7 @@ with pkgs;
             then import test
             else test;
           calledTest = if lib.isFunction loadedTest
-            then callPackage loadedTest {}
+            then loadedTest { inherit pkgs lib; }
```

Comment:
This was supposed to be similar to applying the args in `import ./make-test-python.nix ({ pkgs, ...}: `.
I've added some commits to fix this and test it properly.